src/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <title>THE BEST OF THE BEST</title>
+  </head>
+  <body>
```

Comment:
- Index.html main contain only root element

Reply:
Hello, can you show me at what part of the project description is this listed as a requirement? I've looked at the page and there was no mention of anything except basic HTML markup.
![image](https://user-images.githubusercontent.com/34018257/127254776-464d46f4-97b7-48ef-9c35-64e7ff2d216f.png)

Reply:
In your last point:
![Screen Shot 2021-07-27 at 21 38 54](https://user-images.githubusercontent.com/22258274/127254909-66cf6179-2afc-4d68-82ee-9f8d850aa2a8.png)

Reply:
With this logic, I should also create the entire HTML file inside JS. I don't understand your point, the project IS using ES6, and it's never written that the HTML can ONLY contain the root element.

Reply:
As you can see, Week 2's project clearly states that it should be created dynamically. There is no such requirement listed in this project.
![image](https://user-images.githubusercontent.com/34018257/127256905-51b6d4ca-0851-4785-940b-5fb7c03a9872.png)